README.md

```diff
@@ -27,10 +27,10 @@ npm install -g @asyncapi/generator
 
 ## Development
 
-1. Make sure you have the latest generator installed `npm install -g @asyncapi/generator`.
-1. Modify the template or it's helper functions. Adjust `test/spec/asyncapi.yml` to have more features if needed.
-1. Generate output with watcher enables `npm run develop`.
-1. Open HTML in your browser `open ./test/output/index.html`.
+1. Make sure you have the latest generator installed: `npm install -g @asyncapi/generator`.
+1. Modify the template or its helper functions. Adjust [`dummy.yml`](https://raw.githubusercontent.com/asyncapi/generator/master/test/docs/dummy.yml) to add more features if needed.
```

Comment:
Please add a sentence for contributors to let them know that if they had to modify a dummy file to include more content (to see the feature they worked on in action) they should open a PR to change this fine https://github.com/asyncapi/generator/blob/master/test/docs/dummy.yml in generated repository

Reply:
Added :)